src/components/Input/Input.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+const StyledInput = styled.input`
+  border-radius: 8px;
+  padding: 20px;
+  font-size: 16px;
+  outline: 0;
+  border: 2px solid #eaeaea;
+`;
+
+const Input = () => {
+  return <StyledInput />;
+};
+
```

Comment:
The problem is, that you did not declare any `Props` on `<Input />`. But you want it to have the props a normal `input` tag would have. 
The easiest way to solve this in this situation is to do the following:


```suggestion
const Input = styled.input`
  border-radius: 8px;
  padding: 20px;
  font-size: 16px;
  outline: 0;
  border: 2px solid #eaeaea;
`;

```

this way you are directly exporting the input directly, without another wrapper. 

There are also ways to just pass down props (e.g. in the scenario where you want all the `input` props *plus* some custom ones, but that might be another lesson for the future).

Reply:
Ah okey, thanks, that what we talked about before